pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -434,6 +440,15 @@ func CreateGCECloud(config *CloudConfig) (*GCECloud, error) {
 	}
 
 	gce.manager = &GCEServiceManager{gce}
+
+	// Registering the KMS plugin only the first time.
+	kmsPluginRegisterOnce.Do(func() {
+		// Register the Google Cloud KMS based service in the KMS plugin registry.
+		encryptionconfig.KMSPluginRegistry.Register(KMSServiceName, func(config io.Reader) (envelope.Service, error) {
```

Comment:
I take it, this is the dependency you weren't sure of?  I'm ok with this, I'd also be ok moving out of the cloudprovider proper and doing it inside of a wiring package that knows about both.  Having the dependency go like this is better than the other way around.

Reply:
Agree that this dep is ok; the other way around would not